SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -173,22 +173,20 @@ void USpatialReceiver::OnCommandResponse(const Worker_Op& Op)
 		if (EventTracer != nullptr)
 		{
 			EventTracer->TraceEvent(FSpatialTraceEventBuilder::CreateReceiveCommandResponse(TEXT("SPAWN_PLAYER_COMMAND"), RequestId),
-									Op.span_id, 1);
+									EventTracer->GetSpanForRequestId(CommandResponseOp.request_id).GetConstId(), 1);
 		}
 
 		return;
 	}
 	else if (ComponentId == SpatialConstants::SERVER_WORKER_COMPONENT_ID)
 	{
-		NetDriver->PlayerSpawner->ReceiveForwardPlayerSpawnResponse(CommandResponseOp);
-
 		if (EventTracer != nullptr)
 		{
 			EventTracer->TraceEvent(
 				FSpatialTraceEventBuilder::CreateReceiveCommandResponse(TEXT("SERVER_WORKER_FORWARD_SPAWN_REQUEST_COMMAND"), RequestId),
-				Op.span_id, 1);
+				EventTracer->GetSpanForRequestId(CommandResponseOp.request_id).GetConstId(), 1);
```

Comment:
Do we ever use anything other than `1` for the linear trace Id? We could add a default

Reply:
1 is the count of spans in the span* pointer, I will add your suggestion of commenting the args though so it's easier to spot